### PR TITLE
fixes #225 include ansible_managed in templates.

### LINKED ISSUE #225: Add ansible_managed to templates
Templates should include the {{ ansible_managed }} info.

### PR DESCRIPTION
**Pull request by rwese**
Modified   templates/my.cnf.j2
Modified   templates/root-my.cnf.j2
Modified   templates/user-my.cnf.j2

### CODE REVIEW

File: templates/my.cnf.j2 [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
+# Don't edit manually!
+# {{ ansible_managed }}
```
Comment (geerlingguy) [inline templates/my.cnf.j2]: The `{{ ansible_managed }}` line should suffice. Can you change the files to just have two new lines—first:

```
# {{ ansible_managed }}
```

Then a blank newline before the start of the rest of the file? That follows the convention I use in many of my other roles (e.g. https://github.com/geerlingguy/ansible-role-redis/blob/master/templates/redis.conf.j2 or https://github.com/geerlingguy/ansible-role-ntp/blob/master/templates/ntp.conf.j2).

**Comment by rwese** [comment]
Updated. 

You are right of course that was redundant.

**Comment by geerlingguy** [comment]
Looks good now!

**Comment by geerlingguy** [comment]
I merged, but will wait to release until I have a couple other changes ready, just because this will cause everyone who uses the role to see a `changed` in their playbooks since Ansible will be adding the line in (even if they didn't change any other configs). If I'm going to make a slightly-breaking-ish-change like that, I like to group a few of these changes in for a minor release.

**Comment by geerlingguy** [comment]
Thanks again!

**Pull request merged**